### DISCUSSION TITLE
Markdown cell type and cell overflow "tooltip"

### DISCUSSION THREAD

**Discussion by Far0n** [h2oai/wave#1642] [Q&A] [answered]
Can I somehow make the overflow mode "tooltip" work in conjunction with cell type "markdown"? Wave (v0.23.1) seems to do wrap in this case:

![grafik](https://user-images.githubusercontent.com/6949295/192727039-642a4371-c141-4c0b-b876-94cc9c91c02b.png)

Follow-up question: In the screenshot above, we can see that the table didn't grow enough in height in order to be displayed without a vertical scrollbar (even though there is plenty of space left on the card). Can this behavior be tweaked without setting a fixed height for the table?

**Comment by mturoci** [comment, marked as answer]
> Can I somehow make the overflow mode "tooltip" work in conjunction with cell type "markdown"?

Not supported currently, but very simple to add. Please file a feature request.

> Can this behavior be tweaked without setting a fixed height for the table?

My guess is that it's caused by the line wrap. Could you please try without it and see if the scrollbar is still present? If yes, then we need to fix it on our side. If not, setting static height is your only option.

**Reply by Far0n** [reply]
@mturoci the scrollbar vanishes if there is no wrap (that is if I make the col wide enough that no wrapping is applied)
